cryptofeed/exchange/okex.py

```diff
@@ -48,10 +54,20 @@ def _parse_symbol_data(cls, data: list, symbol_separator: str) -> Tuple[Dict, Di
                 instrument_type = 'option'
             if symbol[-4:] == "SWAP":  # BTC-USDT-SWAP
                 instrument_type = 'swap'
+            if symbol.startswith(INDEX_PREFIX):
+                instrument_type = 'index'
             info['instrument_type'][symbol] = instrument_type
 
         return ret, info
 
+    @classmethod
+    def _entry_is_index_symbol(cls, entry) -> bool:
+        """
+        Returns true if the entry is an index symbol. In the OkEX API,
+        index data is returned as an array of strings, while regular symbols are an array of dictionaries
+        """
+        return isinstance(entry, str)
```

Comment:
no other way to detect if index? 😢

Reply:
Once again, if the data object contained the original URL we could detect it pretty easily, but I'd be hesitant to add such a major feature to our fork. Or what do you think?